ChangeLog.d/muladdc-aarch64-memory.txt

```diff
@@ -0,0 +1,4 @@
+Bugfix
+    * Add missing memory constraints in aarch64 inline assembly for
```

Comment:
We probably should combine the two changelog entries that concern the same bug on different architectures.